src/MSALWrapper/AuthFlow/IntegratedWindowsAuthentication.cs

```diff
@@ -17,6 +17,8 @@ namespace Microsoft.Authentication.MSALWrapper.AuthFlow
     public class IntegratedWindowsAuthentication : AuthFlowBase
     {
         private readonly IEnumerable<string> scopes;
+
+        // TODO: We don't currently warn if the account returned from IWA oesn't match the preferred domain.

```

Comment:
```suggestion
        // TODO: We don't currently warn if the account returned from IWA doesn't match the preferred domain.
```

Did you log as a warning?

Reply:
Yes, right now the domain is unused in this auth flow, so if someone was trying to log in with their Torus account, or maybe some other account that IWA would succeed for, if the domain doesn't match the given domain that's likely something worth warning about. We can open a separate bug for this.